src/lib/CofEnv.ml

```diff
@@ -8,10 +8,7 @@ type env =
   { classes : D.dim UF.t;
     (** equivalence classes of dimensions *)
 
-    cof : D.cof;
-    (** a cofibration which is assumed true, but has not been eliminated yet *)
-
-    status : [`Consistent | `Inconsistent];
+    status : [`Consistent of D.cof | `Inconsistent];
```

Comment:
This may technically work, but it seems kind of like a confusing interface? At least please document the meaning of the thing that is returned.